Makefile

```diff
@@ -67,7 +67,11 @@ test: manifests generate fmt vet envtest ## Run tests.
 # Utilize Kind or modify the e2e tests to load the image locally, enabling compatibility with other vendors.
 .PHONY: test-e2e  # Run the e2e tests against a Kind k8s instance that is spun up.
 test-e2e:
-	go test ./test/e2e/ -v -ginkgo.v
+	export KUBECONFIG=$$(pwd)/.output/kubeconfig && \
+	export PATH=$$(pwd)/.output/ovn-kubernetes/bin:$${PATH} && \
+	export REPORT_PATH=$$(pwd)/.output/ && \
+	cd test/e2e && \
+	go test -v --ginkgo.v --ginkgo.junit-report=$${REPORT_PATH}/test-e2e.junit.xml
```

Comment:
locally under CNAO it panics on 10m timeout
lets add `--test.timeout=1h` please ?

Reply:
I will parameterize those, so external projects can use whatever is fit.

Reply:
that's what makes sense, allow consumers to pass arbitrary settings.